.travis.yml

```diff
@@ -1,10 +1,7 @@
 dist: trusty
 sudo: false
 
-language: java
-
-jdk:
-  - oraclejdk8
+language: python
```

Comment:
the `pip` that is installed by default by travis-ci is broken

Reply:
Why? We used it successfully in the past: https://github.com/Kong/kong/blob/0136876743afe02a71146245f81fb06b81ff2ca5/.ci/setup_env.sh#L113

Reply:
The specific error is as follows:
```
pip install --user datadog
dog metric post test_metric 1
/usr/local/lib/python2.7/dist-packages/urllib3/util/ssl_.py:352: SNIMissingWarning: An HTTPS request has been made, but the SNI (Server Name Indication) ex
tension to TLS is not available on this platform. This may cause the server to present an incorrect TLS certificate, which can cause validation failures. Y
ou can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warni
ngs
  SNIMissingWarning
/usr/local/lib/python2.7/dist-packages/urllib3/util/ssl_.py:150: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3
from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more info
rmation, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecurePlatformWarning
```

Reply:
It might be possible to workaround / fix the issue but as we now run cassandra via docker anyways I saw no reason to not just swap out the travis-ci container

Reply:
Ok, sounds good then. Thanks for sharing the context.